alvr/session/src/settings.rs

```diff
@@ -1274,6 +1292,8 @@ Tilted: the world gets tilted when long pressing the oculus button. This is usef
     #[schema(flag = "steamvr-restart")]
     pub emulation_mode: HeadsetEmulationMode,
 
+    pub performance_level: Switch<PerformanceLevelConfig>,
```

Comment:
This definitely requires a help string detailing what each power mode means (ie which range of power levels can each preset allow)